### DISCUSSION TITLE
Media element background audio only?

### DISCUSSION THREAD

**Discussion by musmuris** [CommunityToolkit/Maui#1910] [General]
Hi,
Looking at playing an MP3 in a cross-platform app in the background (and control start/stop/pause programmatically only).  The samples I've found always show it as a view control I think? Is there anyway to use this as an audio player only - or should I look at native solutions for this instead?

**Comment by bijington** [comment]
I really should know whether it's possible via the toolkit. @jfversluis probably knows best. 

If it doesn't work then you can always use this https://github.com/jfversluis/Plugin.Maui.Audio

**Reply by musmuris** [reply]
Thanks .  I've realised I probably need to play it in a background service as well so that plugin may be easier!  Or exoplayer direct for the android version perhaps?

**Reply by bijington** [reply]
I've seen mention of playing in the background through this toolkit too, although I am struggling to find the detail right now.

**Reply by jfversluis** [reply]
This one I think: https://github.com/CommunityToolkit/Maui/pull/1782

**Reply by ne0rrmatrix** [reply]
Yes that PR which has been approved is just finalizing the changes to doc's and should be ready soon. It will add the required background support for audio or video to play in background while you are running other apps. Controls can be accessed through pull down menu in notification tray.

**Comment by CliffAgius** [comment]
Closed as answered, if closed in error please let us know to re-open.
